tasks/update-infra-deployments/update-infra-deployments.yaml

```diff
@@ -313,12 +321,17 @@ spec:
             infra_pr = github_app.create_mr()
             if "url" not in infra_pr:
                 infra_pr = github_app.get_pr()
-            description = infra_pr["body"]
-            if description == None:
-                description = "Included PRs:"
-            new_pr_link = github_app.get_pr_url_from_sha(revision)
-            new_description = f"{description}\r\n- {new_pr_link}"
-            github_app.update_mr_description(infra_pr["url"], new_description)
+            if "body" in infra_pr:
+                description = infra_pr["body"]
+                if description == None:
```

Comment:
Nit: Should be `if description is None:`. See here: https://peps.python.org/pep-0008/#programming-recommendations

Reply:
Also, so even if there is "body" it can still be None? Can that happen? In that case I would propose:
```python
if infra_pr.get("body") is not None:
    description = infra_pr["body"]
else:
    description = "Included PRs:"
```

Reply:
Fixed in https://github.com/redhat-appstudio/release-service-catalog/pull/260

Reply:
Thanks.